### PR TITLE
docs(storybook): story for PersonSelect selectedPerson attribute

### PR DESCRIPTION
**Pull request by eikeland**
# Component Development

Ref [AB#47637](https://statoil-proview.visualstudio.com/3dadd756-da44-4bb3-8874-330932214dff/_workitems/edit/47637)

Adds storybook entry for selectedPerson. Remove unused attributes from docs.

### Type

- [ ] Feature
- [ ] Fix
- [ ] Hotfix (should release ASAP)
- [ ] Maintenance (deps only)

#### Create User Story(DevOps) or Issue(Github)

- [ ] Describe functional requirements
- [ ] Design plan/suggestion (e.g. images, link to eds design or relevant figma links, mention of eds tokens to use)
- [ ] Communication with UX *(evaluate if needed)*

#### Development

- [ ] Describe the component
- [ ] Code funtional requirements
- [ ] Create stories
- [ ] Create snapshots *(evaluate if needed)*
- [ ] Remove unused imports and commented code
- [ ] Lint code with EsLint (and follow best practises for [code quality](https://docs.fusion-dev.net/development/frontend/code-quality/)
  
#### Create Pull Request

- [ ] Assign relevant reviewers in Github and Cromatic
- [ ] Inform colleagues of pending reviews. We use Teams for this purpose, under fusion-frontend team please tag involved reviewers in the [Code review channel](https://teams.microsoft.com/l/channel/19%3a056a9ba8d8d84a058b24762f85c603ae%40thread.tacv2/Code%2520%2520review?groupId=9589ba16-68a5-47e6-97d7-7d0be0e0d3cb&tenantId=3aa4a235-b6e2-48d5-9195-7fcf05b459b0)

#### Review of Pull Request

- [ ] Follow up on comments from code review and implement necessary changes *(evaluate if needed)*
- [ ] Review approved in github (and chromatic)

#### Publish Code

- [ ] Increment version number
- [ ] Merge PR
- [ ] Consider letting fusion developers know this this shiny new thing exists

### CODE REVIEW

File: storybook/src/stories/person/select-component.stories.mdx [hunk 1]
```diff
@@ -12,23 +12,18 @@ import PersonSelect from '@equinor/fusion-react-person/src';
 
 Name                      | Type                        | Default          | Description
 ------------------------- | -------------------------   | --------------   | -----------
-`placeholder`             | `string`                    | `/`              | TextInput placeholder string
-`initialText`             | `string`                    | `/`              | First item in dropdown before any search is done
```
Comment (asbjornhaland) [inline storybook/src/stories/person/select-component.stories.mdx:16]: `initialText` is removed. I believe it should still be described in the table.

File: storybook/src/stories/person/select-component.stories.mdx [hunk 2]
```diff
@@ -39,71 +34,34 @@ Name                      | Type                        | Default          | Des
...
 
 ## Example
 
-<StoryExample storyId="examples-person-person-select--component" showSource />
+<StoryExample storyId="examples-person-person-select--component" />
 
 ## PersonProvider
 
-In order to use PersonSelect component you need to have [**PersonProvider**](?path=/docs/data-person-provider--page) wrapper.
+The PersonProvider is provided by the Portal or CLI.
```
Comment (asbjornhaland) [inline storybook/src/stories/person/select-component.stories.mdx:49]: Should we clarify that it is the `"Fusion Classic"` portal?

File: storybook/src/stories/person/select-component.stories.mdx [hunk 3]
```diff
@@ -39,71 +34,34 @@ Name                      | Type                        | Default          | Des
...
 
-### Properties/Attributes
 
-Name                      | Type                            | Description
-------------------------- | -----------------------------   | -----------
-`resolve`                 | `PersonResolver`                | Person resolver for a provider. 
+## Controlled component - selectedPerson
 
-### Methods in provider for PersonSelect
+To set selected person in component use the `selectedPerson` attribute.
```
Comment (asbjornhaland) [inline storybook/src/stories/person/select-component.stories.mdx:56]: prop or attr?

File: storybook/src/stories/person/select-component.stories.mdx [hunk 4]
```diff
@@ -39,71 +34,34 @@ Name                      | Type                        | Default          | Des
...
 
-Name                                    | Return Type                     | Description
---------------------------------------  | -----------------------------   | -----------------------------
-`search()`                              | `PersonSearchResult**`          | Returns `PersonSearchResult` for unique search.
+You can select a person by supplying a persons `azureId` or `upn` (email) address or a complete `PersonInfo` object.
 
-Read on how to create your own ***custom search filter*** in [PersonProvider - Custom search](?path=/docs/data-person-provider--page#custom-search)
+To clear a selected person, supply `null` as value. This will also clear any future manual selection from the interface.
```
Comment (asbjornhaland) [inline storybook/src/stories/person/select-component.stories.mdx:60]: Can we rephrase this? It's a bit unclear what the functionality is.

File: storybook/src/stories/person/select-example.stories.tsx [hunk 5]
```diff
@@ -1,23 +1,17 @@
...
-      description: 'HTML ID for component',
-    },
     placeholder: {
       type: 'string',
       description: 'TextInput placeholder string',
     },
-    initialText: {
-      type: 'string',
-      description: 'First item in dropdown before any search is done',
-    },
```
Comment (asbjornhaland) [inline storybook/src/stories/person/select-example.stories.tsx:20]: same as above? initial text should be included?

**Review by asbjornhaland — CHANGES_REQUESTED** [review]

File: storybook/src/stories/person/select-component.stories.mdx (hunk 1 above)
Comment (eikeland) [reply storybook/src/stories/person/select-component.stories.mdx:16]: initialText is now removed in example instead, its not used in component.

File: storybook/src/stories/person/select-component.stories.mdx (hunk 3 above)
Comment (eikeland) [reply storybook/src/stories/person/select-component.stories.mdx:56]: going with property

File: storybook/src/stories/person/select-example.stories.tsx (hunk 5 above)
Comment (eikeland) [reply storybook/src/stories/person/select-example.stories.tsx:20]: initialText is a unused property.

**Review by asbjornhaland — APPROVED** [review]

**Pull request merged**